addons/material_maker/engine/gen_base.gd

```diff
@@ -264,7 +264,7 @@ func get_shader_code(uv : String, output_index : int, context : MMGenContext) ->
 		print(rv)
 	return rv
 
-func _get_shader_code(__, __, __) -> Dictionary:
+func _get_shader_code(uv, output_index, context) -> Dictionary:
```

Comment:
Prefix unused parameters with an underscore to avoid GDScript warnings:

```suggestion
func _get_shader_code(_uv, _output_index, _context) -> Dictionary:
```

Reply:
ah, thanks